### ISSUE TITLE
logo 

### ISSUE THREAD

**Issue by bung87** [bung87/scorper#13]
simple clean sharp logo.   suit with dark and light background.
need artist help!

**Comment by ITwrx** [comment]
i might be able to help facilitate this.

Is the meaning of scorper "a precision chisel" or similar? as in, this framework is like a precision tool? 

Also do we need the name in the logo, or just a graphical icon to represent the idea, and the name can be added beside/on top of/on bottom of the logo as the medium dictates?

**Comment by ITwrx** [comment]
also, do you have a preference for all black and white (and shades of grey), or including color?

**Comment by bung87** [comment]
that's woderfull ! the meaning is right , I imaging the graph similar with "Swiss Army knife", we need both, icon and name provide as separate file, and the combination (icon left + name right ) ( icon top + name bottom), am also developing a static side generator the color schema same as nim official site , so the dark and light background is same , also will be used in readme file with pure white background.
,

**Comment by ITwrx** [comment]
Well a Scorper has a different meaning than a Swiss Army knife. a Swiss Army knife means it has everything included and can be used for anything, but a Scorper is a precision (and singular) tool used for one (general/type of) thing (wood engraving, or in this case, web applications). In other words, it's a precision tool to make what you want, but doesn't include everything by default like a Swiss Army knife. This also more closely matches the idea of a micro framework. A Swiss Army knife says "full-fledged framework" or "everything included but the kitchen sink". Also, the Swiss Army knife idea is a little "well used", no offense. That may not be as well known to you since English is not your native tongue. 

I already had an idea for the logo that more closely matches the meaning of Scorper, and also (coincidentally/luckily and gracefully) alludes to the capabilities/opportunities the framework (presumably, at this stage) gives, but i was going to surprise you, as the idea is part of the logo.  If you are brave enough to let us try, I think you will have an original and creative logo that you will like. :)

I am planning to fund it's creation as a donation, and for attribution for me and the artist, if you are ok with me releasing it to you under the cc-by-sa 4.0 license. https://creativecommons.org/licenses/by-sa/4.0/

We don't have any specific requirements on the prominence of the attribution, except that if you make a site for Scorper, anywhere people can see it on there would be fine, and a note in the readme of the project repo. So greedy, i know. :)

What do you think? Are you game?

**Comment by bung87** [comment]
yeah , " it's a precision tool" , I imaging it's one tool of  "Swiss Army knife" , but I dont know the exactly sharp , just iron sharped and handle or something just like  "Swiss Army knife" gives me, it's always be a micro framework, won't go full stack path, I agree with attribution of you and the artist, if you already have written it in markdown format , please also provide here or create a PR , I thought add you to sponsors list in readme. Thanks again! I feel not alone with this project : )

**Comment by bung87** [comment]
the scorper may have a website on GitHub pages that use [crown_ui](https://github.com/bung87/crown_ui)(it needs karax merge my PR first, I only run it locally) am recently developing, I just finished the layout part, haven't design every pages looking.  attribution there in the future is fine to me.

**Comment by bung87** [comment]
btw, the name "scorper" is my friend suggested, I like the alphabets combination and the word pronunciation , also the meaning kind like match my micro framework.

**Comment by ITwrx** [comment]
I like "scorper" too. We will start on the logo. :)

**Comment by ITwrx** [comment]
oh, i just realized what you were trying to say in your last response about the Swiss Army knife. I think trying to show the Scorper as one tool of the Swiss Army knife is not a bad idea, but a little difficult to be clear about, and for people to know what the Scorper is in that image, etc. 

Also, the micro-framework is the Swiss Army knife in the Swiss Army knife metaphor (orm, templating lib, etc. are the tools that can be used under the micro framework, but are not on the same top level as the micro framework), but since Swiss Army knife is typically used for "having everything already included", is not the best metaphor, IMO. 

If it's ok with you, we will create the logo with our idea. I will risk the funding money, and if you don't like the result, you don't have to use it. I think you will like it. :)

**Comment by bung87** [comment]
dont make complicated , just one sharp tool that solve problem .

**Comment by ITwrx** [comment]
it won't be complicated.

**Comment by bung87** [comment]
@ITwrx  hey , how's going? I can't find you in any IM platform, so ask here

**Comment by ITwrx** [comment]
All is well. Yeah, i don't usually use proprietary, centralized messaging (surveillance) software (github is grandfathered in, for now) and haven't started to use Matrix due to it's slow, greedy python-based server. Session and Status.im are cool, but i just don't keep them open all the time yet. I have my own xmpp server and use that with Conversations and Gajim. I am planning to make that server available for public accounts soonish. I haven't bothered to federate it either. I'm not a big chatter anyways. :) Speaking of federation, I'm also on the fediverse @ITwrx@blurts.net. blurts.net is not available for public registration, but I am planning on doing that soonish too.

Regarding the logo, I told the artist not to get in a huge rush, but i expect something in the next week or two.

**Comment by bung87** [comment]
okay, so basically you're in private xmpp network, could you provide the artist information , any link ？ I'd like to see some artworks.

**Comment by ITwrx** [comment]
and on the fediverse, which anyone can reach me on by signing up for an account on any fedi server or setting their own up. 

The artist is at https://luciusrafi.com, but he doesn't feature logos on that site as a product or service, as graphic design is not a service he officially offers. I know he did the logo on his site, and the logo for http://EMH.biz, though they were both done a while back.

**Comment by bung87** [comment]
this one ? https://fediverse.party/en/fediverse/ , looks cool, first time heard of such a network, I checked luciusrafi.com as you said he doesn't post his graphic design works, well , he is cool guy. drawing something I dont understand like other artists.

**Comment by ITwrx** [comment]
yes that's a site about the fediverse. here's another article: https://en.wikipedia.org/wiki/Fediverse

**Comment by bung87** [comment]
@ITwrx hey, how is going?

**Comment by ITwrx** [comment]
It's finished! Please see the following images. It's a scorper inside of a powerful and mysterious keyhole. The simple idea being that Scorper is a precision tool that is the key to accessing other powerful and unknown possibilities, realizing potential, etc.  

The attribution text, that should be shown on any site where the images (or their derivatives) are used, should read: "The Scorper logo was created and released to the Scorper project under the CC BY-SA 4.0 by https://ITwrx.org and https://LuciusRafi.com".

Enjoy, and keep up the good work on Scorper!

# Text Below Logo for use with Black Background #
![text_below_dark_bkg](https://user-images.githubusercontent.com/1002762/125472442-e22c6bf1-2e54-44b7-a7b3-39f0f792a70a.png)
-
-
-
-
# Text Below Logo for use with White Background #
![text_below_white_bkg](https://user-images.githubusercontent.com/1002762/125472447-1e08c24c-ff11-4bd4-b2f7-604582d02a35.png)
-
-
-
-
# Text Beside Logo for use with Black Background #
![text_beside_dark_bkg](https://user-images.githubusercontent.com/1002762/125472450-2eb421e0-2090-41b8-bbac-e4358f3e4204.png)
-
-
-
-
# Text Beside Logo for use with White Background #
![text_beside_white_bkg](https://user-images.githubusercontent.com/1002762/125472454-9b35f3ec-ca88-4652-99c5-bb81bcba5bb0.png)
-
-
-
-
# Logo Without Text for use with White or Black Background #
![logo_without_txt](https://user-images.githubusercontent.com/1002762/125472534-431d80e8-1b73-4498-bce7-d2f9f195b8f0.png)
-
-
-
-
# Text Only for use with Black Background #
![text_only_dark_bkg](https://user-images.githubusercontent.com/1002762/125472538-917a27b1-ffce-4221-b694-a7cb18e77677.png)
-
-
-
-
# Text Only for use with White Background #
![text_only_white_bkg](https://user-images.githubusercontent.com/1002762/125472542-aa658f55-fd88-4c7d-8b59-b440524868d7.png)
-
-
-
-

**Comment by bung87** [comment]
Great! Thank you and Lucius Rafi , I'v add logo and attribution to readme , good looking now!
